fpga/src/main/scala/arty/HarnessBinders.scala

```diff
@@ -100,3 +101,83 @@ class WithArtySPIFlashHarnessBinder extends OverrideHarnessBinder({
     }
   }
 })
+class WithArtyGPIOHarnessBinder extends OverrideHarnessBinder({
+  (system: HasPeripheryGPIOModuleImp, th: ArtyFPGATestHarness, ports: Seq[GPIOPortIO]) => {
+    withClockAndReset(th.harnessClock, th.harnessReset) {
+      implicit val p: Parameters = GetSystemParameters(system)
+
```

Comment:
Suggestion: Add a require here to check that the underlying system was configured with GPIO width of 32.